test/fixtures/test/states.json

```diff
@@ -2,7 +2,7 @@
   {
     "id": "22222",
     "name": "To Do",
-    "sequence": 1,
+    "sequence": 100,
```

Comment:
Changed these to match what’s returned by the api in production.

Those sequence values are `100`, `200`, `300`, etc. instead of `1`, `2`, `3`, etc.